test/test_jit.py

```diff
@@ -15106,8 +15106,8 @@ def test_functional_multi_head_attn_fast_path_None(self):
         with torch.no_grad():
             fn_out = torch.nn.functional.multi_head_attention_forward(query, key, value,
                                                                       embed_size, nhead,
-                                                                      None,
-                                                                      None,
+                                                                      multi_head_attn_nn.in_proj_weight,
+                                                                      multi_head_attn_nn.in_proj_bias,
```

Comment:
What is this change? Is this to avoid the assertion that you added elsewhere to prevent bias from being None?

Projection bias can be None, see the test and fix in this as per issue raised. https://github.com/pytorch/pytorch/pull/88970